src/pages/Event/Event2023FallMissions.tsx

```diff
@@ -1,4 +1,8 @@
-import { memo } from "react";
+import { memo, useMemo } from "react";
+
+import { Quest } from "types/event2023fall";
```

Comment:
type-only import 로 수정하시면 좋을 것 같아요

```suggestion
import type { Quest } from "types/event2023fall";
```

참고 : https://kagrin97-blog.vercel.app/types/TypeOnly-Import,Export